### PR TITLE
Invoke cluster operations on jobs.clusterexec.

### PR DESCRIPTION
**Pull request by mbarnes**
Still debugging, but I think this is all we need to connect up to clusterexec.

Requires: https://github.com/projectatomic/commissaire-service/pull/27

### CODE REVIEW

**Comment by mbarnes** [comment]
@ashcrow You can review this but don't merge just yet.  Still trying to confirm this works.

File: src/commissaire_http/handlers/clusters/operations.py [hunk 1]
```diff
@@ -254,13 +257,8 @@ def create_cluster_operation(model_cls, message, bus):
...
 
-        # TODO: Hook up service call
-        # if isinstance(model_cls, models.ClusterUpgrade):
-        # elif isinstance(models_cls, models.ClusterRestart):
-
-        result = bus.request(
-            'storage.save', params=[
-                model_cls.__name__, model.to_dict()])
+        # XXX Assumes the only method argument is cluster_name.
+        result = bus.request(routing_key, params=[cluster_name])
```
Comment (ashcrow) [inline src/commissaire_http/handlers/clusters/operations.py]: nit: There is a chance``RemoteProcedureCallError`` could be raised upon issues.

Comment (mbarnes) [inline src/commissaire_http/handlers/clusters/operations.py]: That's handled a few lines down.  Similarly in `create_cluster_deploy()`.

**Comment by mbarnes** [comment]
@ashcrow This is ready for re-review, since https://github.com/projectatomic/commissaire-service/pull/27 got merged.

**Comment by ashcrow** [comment]
@mbarnes reviewing now.

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
LGTM

**Pull request merged**